tensorrt_llm/_torch/memory_buffer_utils.py

```
@@ -0,0 +1,97 @@
+#   [t4] A = torch.zeros(....) -> allocate buffer A in allocator but not use memory in cudagraph pool
+#        OOM may happen
+# NOTE: it requires all tensors with the same identifier (buffer_name here) have the same dtype. Will
+#       upgrade this.
+class Buffers:
+
+    def __init__(self):
+        self.allocated_buffer_in_graph_pool: dict[str, list[torch.Tensor]] = {}
+        self.allocated_buffer_in_runtime: dict[str, torch.Tensor] = {}
+
+    def get_buffer(self, tensor_shape: list[int], dtype: torch.dtype,
+                   buffer_name: str, create_if_miss: bool, pin_memory: bool):
```

Ruff (0.12.2)

Unused method argument: `create_if_miss`

(ARG002)